CHANGELOG.md

```diff
@@ -56,6 +56,9 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
   to add convenient functions that compute information based on the record itself and to provide better documentation.
   ([\#2340](https://github.com/seqan/seqan3/pull/2340), [\#2380](https://github.com/seqan/seqan3/pull/2380),
   [\#2389](https://github.com/seqan/seqan3/pull/2389))
+* Deprecated `seqan3::field::seq_qual`. Use `seqan3::fiel::seq` and `seqan3::field::qual` instead.
+  ([\#2379](https://github.com/seqan/seqan3/pull/2379)). Checkout [SeqAn3 Cookbook - Write Record](https://docs.seqan.de/seqan/3.0.2/cookbook.html)
+  for usage.
```

Comment:
```suggestion
* Deprecated `seqan3::field::seq_qual`. Use `seqan3::fiel::seq` and `seqan3::field::qual` instead.
  ([\#2379](https://github.com/seqan/seqan3/pull/2379)). Check out 
  [SeqAn3 Cookbook - Write Record](https://docs.seqan.de/seqan/3.0.2/cookbook.html)
  for usage.
```

Isn't it wrong to link to the `3.0.2` ? Because this is the old way to do it.
I guess the correct way would be to use `3.0.3` and set the server up to redirect to the `master` documentation until `3.0.3` is released?

cc @marehr

Reply:
I set up a symlink, so we can just use `3.0.3`